### DISCUSSION TITLE
App not visible in Health App on Simulator

### DISCUSSION THREAD

**Discussion by jonahschlie** [agencyenterprise/react-native-health#265] [Q&A]
I created a react native app and want to get access to the health data i typed in manually in health app. But after i tried initHealthKit my app is not shown in the health app as an app that want acess to the data. any tips how i can manage that?

**Comment by GGGava** [comment]
Hello, have you provided any data types in the `permissions` parameter when calling `initHealthKit`? You must send the data types you want to access.

Example:

```javascript
/* Permission options */
const permissions = {
  permissions: {
    read: [AppleHealthKit.Constants.Permissions.HeartRate],
    write: [AppleHealthKit.Constants.Permissions.Steps],
  },
} as HealthKitPermissions;

AppleHealthKit.initHealthKit(permissions, (error: string) => {
  /* Called after we receive a response from the system */

  if (error) {
    console.log('[ERROR] Cannot grant permissions!');
  }

  /* Can now read or write to HealthKit */
```
